torch/csrc/autograd/profiler.h

```diff
@@ -180,6 +180,9 @@ struct TORCH_API Event final {
 // a std::vector resize from taking a large amount of time inside
 // a profiling  event
 struct RangeEventList {
+  // This mutex is used to serialize access when different threads are writing
+  // to the same instance of RangeEventList.
+  std::mutex mut;
```

Comment:
nit: we usually call them "mutex_" in the autograd subfolder if they don't have a special name. "mut" looks a bit weird.

Reply:
I changed it to mutex_ in the update.